### PR TITLE
COMMERCE-9454 Order Forecast Chart widget "No Data Available" is not localized

### PR DESCRIPTION
**Pull request by manuelecastro**
[COMMERCE-9454](https://issues.liferay.com/browse/COMMERCE-9454)

### CODE REVIEW

**Comment by manuelecastro** [comment]
ci:test:sf

**Comment by manuelecastro** [comment]
ci:test:sf

**Comment by manuelecastro** [comment]
ci:test:relevant

**Comment by manuelecastro** [comment]
ci:test:commerce

File: modules/dxp/apps/commerce/commerce-dashboard-web/src/main/resources/META-INF/resources/forecast.jsp [hunk 1]
```diff
@@ -51,7 +51,6 @@ if (commerceAccount != null) {
 			accountIds: <%= accountIds %>,
 			categoryIds: <%= categoryIds %>,
 			noAccountErrorMessage: Liferay.Language.get('no-account-selected'),
```
Comment (riccardo-ferrari) [inline modules/dxp/apps/commerce/commerce-dashboard-web/src/main/resources/META-INF/resources/forecast.jsp:53]: Hi @manuelecastro,
Thanks for you fix. I see we likely have the very same issue in this line (no-account-selected). Could you update your PR and fix this too?
You might need to adjust the code in ForecastChart.es.js:48 (yet another issue I think)

Comment (manuelecastro) [reply modules/dxp/apps/commerce/commerce-dashboard-web/src/main/resources/META-INF/resources/forecast.jsp:53]: Hello @riccardo-ferrari ! I made the changes you asked. I'm just waiting for a response from Manuel Molina to see if this is a case for creating a new issue or not. :)

**Comment by riccardo-ferrari** [comment]
Don't worry, just append those commits here. Manuel is aware already

**Comment by manuelecastro** [comment]
reopen at: [COMMERCE-9454](https://github.com/liferay-commerce/liferay-portal/pull/3350)

**Pull request closed without merging**